lib/Elastica/Aggregation/Derivative.php

```diff
@@ -35,7 +35,7 @@ public function setBucketsPath(string $bucketsPath)
      *
      * @return $this
```

Comment:
The `return $this` should be removed too :)

Reply:
Lets separate PR's which are fully automated and manual changes ;-)